.eslintrc.js

```diff
@@ -19,9 +21,9 @@ module.exports = {
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
     'prettier/prettier': 'error',
-    quotes: [2, 'single', 'avoid-escape'],
+    quotes: [2, 'single', 'avoid-escape']
   },
   globals: {
-    LIB_VERSION: true,
-  },
-};
+    LIB_VERSION: true
```

Comment:
there is no global var with this name

Reply:
Added Typescript to stories, sandboxes, tests. Added test for types definition. Removed unused vars and folders